docs(marznode-marzneshin-config): farsi translation

w0l4i opened the pull request:
Updating a bunch of Marzneshin documents in a chore scheduled event.

w0l4i committed:
marzneshin.md

adding update,stop,start variables

w0l4i committed:
marznode.md

Complete Node Installation

w0l4i committed:
Update marznode.md

w0l4i committed:
Update marznode.md

w0l4i committed:
Update hysteria.md

w0l4i committed:
Update hysteria.md

w0l4i committed:
Update hysteria.md

w0l4i committed:
Update hysteria.md

w0l4i committed:
Update marznode.md

w0l4i committed:
Create hysteria_Fa.md

w0l4i committed:
Create marznode_fa.md

w0l4i committed:
Update marznode_fa.md

w0l4i committed:
Create marznehsin_fa.md

w0l4i committed:
Create codebase_fa.md

w0l4i committed:
Create donation_fa.md

w0l4i committed:
Create api_fa.md

w0l4i committed:
Create _index_fa.md

w0l4i committed:
Create marzneshin_fa.md

w0l4i committed:
Create marznode_fa.md

w0l4i committed:
Create basic-install_fa.md

w0l4i committed:
Create concepts_fa.md

w0l4i committed:
Delete content/docs/getting-started/advanced-install/concepts_fa.md

w0l4i committed:
Create concepts_fa.md

w0l4i committed:
Create _index_fa.md

erfjab commented on content/docs/configuration/marznode.md:
````diff
@@ -1,4 +1,93 @@
+1. Installing Docker and pulling Marznode with this command:
+
+```bash
+apt-get update -y && apt-get upgrade -y && curl -fsSL https://get.docker.com | sh && git clone https://github.com/khodedawsh/marznode && cd marznode && docker compose up -d
````
why not marzneshin? 
`clone https://github.com/khodedawsh/marznode`

realartin replied:
done

w0l4i committed:
Update marznode.md

clone https://github.com/khodedawsh/marznode to clone https://github.com/marzneshin/marznode

w0l4i committed:
Update marznode_fa.md

w0l4i reviewed (commented):
ok

Krr0ptioN commented on content/docs/configuration/marzneshin.md:
```diff
@@ -84,3 +84,10 @@ RECURRENT_NOTIFICATIONS_TIMEOUT=
+
+| Variable | Description                                                                                 |
+| -------- | ------------------------------------------------------------------------------------------- |
+| marzneshin update     | you can update Marzneshin easily with |
```
Update Marzneshin

w0l4i committed:
Rename marznehsin_fa.md to marznehsin.fa.md

w0l4i committed:
Rename marznode_fa.md to marznode.fa.md

w0l4i committed:
Rename hysteria_Fa.md to hysteria.fa.md

w0l4i committed:
Rename api_fa.md to api.fa.md

w0l4i committed:
Rename codebase_fa.md to codebase.fa.md

w0l4i committed:
Rename _index_fa.md to _index.fa.md

w0l4i committed:
Rename _index_fa.md to _index.fa.md

w0l4i committed:
Rename marzneshin_fa.md to marzneshin.fa.md

w0l4i committed:
Rename marznode_fa.md to marznode.fa.md

w0l4i committed:
Rename basic-install_fa.md to basic-install.fa.md

w0l4i committed:
Rename concepts_fa.md to concepts.fa.md

realartin commented:
renaming .fa.md files for _index.fa
